clarity/src/vm/docs/mod.rs

```diff
@@ -2773,10 +2773,18 @@ mod test {
             u32::MAX
         }
 
+        fn get_v3_unlock_height(&self) -> u32 {
+            u32::MAX
+        }
+
         fn get_pox_3_activation_height(&self) -> u32 {
             u32::MAX
         }
 
+        fn get_pox_4_activation_height(&self) -> u32 {
```

Comment:
Is there a purpose to having different "get_*_unlock_height" ? they all return the same u32::MAX? Should this be reduced to just one function "get_unlock_height" ? Similarly wondering about activation height.